packages/components/src/Emphasis/Emphasis.test.js

```diff
@@ -66,4 +66,20 @@ describe('Emphasis', () => {
 		// then
 		expect(wrapper.find('em').length).toBe(1);
 	});
+
+	it('should emphasize multiple words', () => {
+		// given
+		const wrapper = shallow(<Emphasis {...props} value={['lazy', 'fox', 'dog']} />);
```

Comment:
please use rtl to add tests.
enzyme is deprecated

Reply:
done 👍